.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
It’s not appropriate to include this directive in shared version control. You can’t assume that everyone will want to automatically use the flake, and certainly can’t assume that every user of direnv is also using Nix.